src/main/java/org/littil/api/ApplicationLifeCycle.java

```diff
@@ -40,12 +40,18 @@ public class ApplicationLifeCycle {
             "testschool2@littil.org", createLocationForDevUser("Marco Pololaan 6", "3526GJ"), //
             "testschool3@littil.org", createLocationForDevUser("Sint Jansstraat 4", "9712JN") //
     );
+    private static final String ZIPCODE_REGEX = "^[1-9][0-9]{3}[A-Z]{2}$";
+    private static final String EMAIL_REGEX = "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$";
```

Comment:
Do we need an e-mail regex if the users are only added if the e-mail exists in Auth0? Is there perhaps a validity check in Auth0 already?

Reply:
validatie voor email adres verwijderd. Wordt al afgevangen in de frontend bij het registreren.